apps/montreal_elixir_web/lib/montreal_elixir_web/channels/twitter_timeline_channel.ex

```diff
@@ -0,0 +1,21 @@
+defmodule MontrealElixirWeb.TwitterTimelineChannel do
```

Comment:
Should we have this in a name space `MontrealElixirWeb.Twitter.TimelineChannel` the file could just be named `twitter.timeline_channel.ex`? We could use the same naming convention for the listener.